specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-02-02-preview/managedClusters.json

```diff
@@ -5528,6 +5532,20 @@
       },
       "description": "Azure Defender settings for the security profile."
     },
+    "AzureKeyVaultKms": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable Azure Key Vault key management service. The default is false."
+        },
+        "KeyId": {
+          "type": "string",
+          "description": "Identifier of Azure Key Vault key. When Azure Key Vault key management service is enabled, this field is required and must be a valid key identifier. When Azure Key Vault key management service is disabled, leave the field empty."
```

Comment:
This this an ARM ID, or something else? I think you're talking about: `https://{vault-name}.vault.azure.net/{object-type}/{object-name}/{object-version}`. If so, possibly include that in the docs so people know the format they should be providing here. Since it might be more complicated because managed HSM seems to have a different format you could also just link to the KV docs: https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates#vault-name-and-object-name - or if there's a better doc to link to than that feel free to use it instead.